config/nais.io_applications.yaml

```diff
@@ -568,6 +568,11 @@ spec:
               properties:
                 enabled:
                   type: boolean
+                mounts:
+                  properties:
+                    default:
+                      type: boolean
+                  type: object
```

Comment:
Not sure about this one. The motivation behind this is to accomodate future planned abstractions to help the developers avoid having to define `paths` manually for mounting Vault secrets, e.g.:

```yaml
spec:
  vault:
    enabled: true
    mounts:
      default: true
      apigw:
        - api: <api>
          consumer: <consumer>
      serviceuser: 
        - name: <name>
      ... 
```

The option affects the behavior of `spec.vault.paths` and it would conceptually make sense to include there, however that isn't possible without introducing breaking changes.

Proposed alternatives:

1. Rename to `spec.vault.keepDefaultPath`? Feels kind of disconnected from `paths`.
2. Rename to `spec.vault.loadMounts.defaults`? Slightly more verbose, but more descriptive?